eberron.md

```diff
@@ -113,22 +123,21 @@ When cast at 6th level or above, the weapon gains Reach 2.
 - **Range:** self
 - **Duration:** 10 minutes.
 
-Cast against 2 terrible dice, targeting an unconscious creature. The creature rises in a stupor and are no longer incapacitated. While in this state they keep their temporary wounds and continue to roll death saving throws. They cannot take any action except movement and attacks, and any turn they do not take damage, are attacked, or attack another creature they automatically fail their death saving throw. The spell ends if they regain enough wounds to regain consciousness.
+Cast against 2 terrible dice, targeting an unconscious creature. The creature rises in a stupor and are no longer incapacitated. While in this state they keep their temporary wounds and continue to roll death saving throws. They cannot take any action except movement and attacks, and any turn they do not take damage, are attacked, or attack another creature they automatically fail their death saving throw. While under this effect they ignore the first wound of damage they take from attacks each round. The spell ends if they regain enough wounds to regain consciousness.
```

Comment:
After talking with you and Zack it seems raising your friends to die for realsies wasn't the most attractive. This will at least give them some survivability benefit.